go-update/src/main/java/org/reactome/release/goupdate/DuplicateReporter.java

```diff
@@ -81,23 +84,29 @@ public DuplicateReporter(MySQLAdaptor adaptor)
 			{
 				for (GKInstance i : instances)
 				{
-					for (GKSchemaAttribute referrerAttrib : (Set<GKSchemaAttribute>)i.getSchemClass().getReferers())
-					{
-						@SuppressWarnings("unchecked")
-						Collection<GKInstance> referrers = (Collection<GKInstance>) i.getReferers(referrerAttrib);
-						if (referrerCounts.containsKey(i.getDBID()))
-						{
-							referrerCounts.put(i.getDBID(), referrerCounts.get(i.getDBID()) + referrers.size());
-						}
-						else
-						{
-							referrerCounts.put(i.getDBID(), referrers.size());
-						}
-					}
+					Long dbid = i.getDBID();
```

Comment:
Better to use primitives whenever possible

Reply:
`getDBID` returns a `Long` not a `long`.

Reply:
Yeah, I saw that.  I tried out `long dbid = i.getDBID()` and it will auto unbox for you.  It's just better to have variables that use the value stored in a primitive so there's never concern about value vs reference equality issues.